README.md

```diff
@@ -1 +1,67 @@
-# pointboard
+# Point Board
```

Comment:
Maybe it would be nicer to include a brief introduction to this project, so that as a new user I would know what to expect for this.

Since you use Dockerfiles I think it might not be as important to introduce how to generate template and install dependencies. Maybe just a short one for how to test if the micro services apps work locally?

I think if this project is more like a useful tool then i would expect it to be deployed easily and don't need to install those dependencies (fully dockerized). If it is more an example showing ppl how to use Copilot, then I would expect more introduction for other features in Copilot like how do you  check if your application is running healthy. How to check the logs. Do we want to create a pipeline for continuous delivery etc.